zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -865,6 +865,30 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
     }
   };
 
+  $scope.parseTableCell = function(cell) {
+    //here we try to automatically parse the data type for each
+    //cell in the table, i.e. string/number/date
+    // 1. is this a date?
+    var supportedDateFormats = ['MM-DD-YYYY', 'DD-MM-YYYY', 'MM/DD/YYYY',
```

Comment:
That function is used to test whether an object is JS date object and cannot be used here.
We are basically dealing with strings.